### ISSUE TITLE
Download has no audio...

### ISSUE THREAD

**Issue by maleghast** [bobbicodes/ytdl#1]
Hi :-)

I love the potential utility, and I love seeing nbb stuff / tools, but when I used this earlier I got video without audio.  I tried it twice, no luck...

Am I doing something wrong, or is this a bug..? (I used the example command from the README)

Thanks!

**Comment by bobbicodes** [comment]
I wonder if it could be related to this: https://github.com/fent/node-ytdl-core/issues/542

**Comment by bobbicodes** [comment]
Someone said this has to do with higher resolutions being delivered in separate streams. I noticed that while mine had audio, it had pretty low resolution. I can probably look into what options are available for selecting streams but it may be necessary to download audio and video separately and combine them using ffmpeg.

I created this repo based on a gist I made a couple of years ago that used fluent-ffmpeg, another node package: https://gist.github.com/bobbicodes/9b3e6dfb6e268c84f5058a8e0f02179a

My use case at the time was using the audio filters to create aggressively compressed versions of videos, since I have sensory processing issues.

Anyway, the gist shows that the two projects indeed play well together, so it could just be a matter of providing the right combination of options.
